src/Main.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import HornedBeast from './HornedBeast.js';
+
+class Main extends React.Component {
+  render () {
+    return (
+      <>
+        <HornedBeast name="HB1" imageUrl="srcurl"/>
+        <HornedBeast name="HB2" imageUrl="srcurl"/>
+      </>
+    )
+  }
+}
```

Comment:
Awesome job ! 👍 

You are really close here :) We need to pass the following to the `<HornedBeast />`:
 - `title`, `image`, and `description`. Update name and imageUrl to match instructions for lab 👍